app/views/component/locations/detail/_body.html.haml

```diff
@@ -345,7 +344,15 @@
     let serviceCardElement = document.getElementById(`service-card-${i}`);
     serviceCardElement.classList.toggle("collapsed");
 
-    e.target.setAttribute("aria-expanded", serviceCardElement.classList.contains("collapsed")? "false" : "true");
+    e.target.setAttribute("aria-expanded", e.target.classList.contains("closed")? "false" : "true");
```

Comment:
This line doesnt seem to be working, the `aria-expanded` attribute remains true even when the section is collapsed


Uploading Screen Recording 2024-12-20 at 3.11.48 PM.mov…

Reply:
Good catch @anna-dorigo! I just pushed a fix!